project/app/api/predict.py

```diff
@@ -22,21 +22,10 @@ class Item(BaseModel):
     src: str = Field(..., example='"www.twitter.com"')
     date: str = Field(..., example= "2020-05-30")
```

Comment:
What happened to the placeholder feature/field type instructions we made as instructions on the Fast API? Does that live somewhere else?

Reply:
Sorry, I'm having a hard time pinpointing your train of thought. Can you elaborate on this?